src/logic/user-items-filter.ts

```diff
@@ -0,0 +1,150 @@
+import { Avatar, Outfit, Outfits } from '@dcl/schemas'
+import { BaseComponents, OnChainEmote, OnChainWearable } from '../types'
+import { splitUrnAndTokenId } from './utils'
+
+export type UserItemsFilter = {
+  filterNotOwnedItemsFromAvatars(avatar: Avatar[], owner: string): Promise<Avatar[]>
```

Comment:
perhaps is better to move the `filterNotOwnedItemsFromAvatars` function to profiles and the other one to outfits?

Reply:
`getSanitizedAvatars` really was my first name here, but then I realized that `sanitizes` doesn't really explain what we are doing. What about renaming it to `removeNotOwnedItemsFromAvatars`?

Regarding to move these functions to outfits and profiles, it is also something I thought about but the thing is `logic/profiles.ts` is growing fast and it already contains 13 helper/utility functions which is making harder to fully understand the responsibility of the module (at least IMO). I don't see any concerns about moving the other function to `outfits`. What do you think?

Reply:
well, I understand your concern but I don't think that should stop us from adding the function there, Since this is part of profiles, moving it somewhere else is not really helping to reduce the responsibility, it's just another file you have to read.